activities/tool-management/mattermost.md

```diff
@@ -12,7 +12,23 @@ Mattermost is a flexible, open source messaging platform that enables secure tea
 
 ## Mattermost management
 
-Mattermost is installed, maintained and kept up-to-date by the Operations coordinator. User registration and management also falls under the Operations coordinator responsibilities. This includes uploading important emojis :)
+Mattermost is installed, maintained and kept up-to-date by the Operations coordinator and is running on a VPS with 8 GB of RAM. User registration and management also falls under the Operations coordinator responsibilities. This includes uploading important emojis :)
```

Comment:
Is it necessary to repeat in so many places that the Operations Coordinator is responsible for managing the individual tools? I think an easier place would be to have this as a paragraph in the index. We can use the the text from section 3. in the Operations Handbook.

Reply:
I added the paragraph, but removing it everywhere is also not useful, especially if someone is getting to the page outside the index, or if we ever want to highlight a particular tool, etc.